pep-0675.rst

```diff
@@ -913,6 +960,246 @@ is documentation, which is easily ignored and often not seen. With
 ``Literal[str]``, API misuse requires conscious thought and artifacts
 in the code that reviewers and future developers can notice.
 
+.. _appendix_C:
+
+Appendix C: ``str`` methods that preserve ``Literal[str]``
+==========================================================
+
+The ``str`` class has several methods that would benefit from
+``Literal[str]``. For example, users might expect
+``"hello".capitalize()`` to have the type ``Literal[str]`` similar to
+the other examples we have seen in the `Inferring Literal[str] section
+<inferring_literal_str>`_ section. Inferring the type ``Literal[str]``
+is correct because the string is not an arbitrary user-supplied
+string. In other words, the ``capitalize`` method preserves the
+``Literal[str]`` type. There are several other methods that preserve
+the ``Literal[str]``.
```

Comment:
```suggestion
``Literal[str]``.
```

The definite article requires "type" afterward, which would be repetitive, so I just removed it